vault/request_handling.go

```diff
@@ -543,6 +543,9 @@ func (c *Core) handleCancelableRequest(ctx context.Context, req *logical.Request
 				}
 				break
 			}
+			if token == nil {
+				return logical.ErrorResponse("bad token"), logical.ErrPermissionDenied
```

Comment:
maybe `invalid token` might be better? cause browsing through current error responses, invalid seems to be the word mostly used for erroneous input.

Reply:
Hi, I just committed a change to the error message, changing it to invalid token. I accidentally wrote this as 'null token' in the commit message, but in the code it is 'invalid token' as requested.